components/filter/filter.js

```diff
@@ -397,6 +409,20 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		this._dispatchChangeEvent(dimension, { valueKey: valueKey, selected: selected });
 	}
 
+	_handleClear() {
+		const dimension = !this._activeDimensionKey ? this._dimensions[0] : this._dimensions.find(dimension => dimension.key === this._activeDimensionKey);
```

Comment:
I see this code repeated... might be worth throwing it in a `_getActiveDimensionOrFirst()` or something.